plugins/rtmp-services/data/schema/service-schema-v5.json

```diff
@@ -233,7 +233,7 @@
                     },
                     {
                         "$comment": "Require recommended output field if protocol field is not RTMP(S)",
-                        "if": { "required": ["protocol"], "properties": { "protocol": { "pattern": "^(HLS|SRT|RIST|WHIP)$" } } },
+                        "if": { "required": ["protocol"], "properties": { "protocol": { "pattern": "^(HLS|SRT|RIST)$" } } },
```

Comment:
This removes the requirement of the `recommended` field for services using WHIP. Why?

Reply:
Never mind. I have re-read the change and the PR description. It's removing the requirement for the `output` key within the `recommended` key. I would have to investigate more to better understand what is "best" here.

Reply:
I mentioned that as point #2 in my description - that field is deprecated and there's no appropriate value for WHIP services. The existing options are:

```
"rtmp_output",
"ffmpeg_hls_muxer",
"ffmpeg_mpegts_muxer"
```

WHIP is none of those. I could certainly add a value called `whip_output` in there, but it would be an additional enum on a deprecated field used by nothing and read by nobody. What's the point?

Reply:
Oop, missed your reply, all good :)